### PR TITLE
Include all source files

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def build_extensions(self):
             pass
         build_ext.build_extensions(self)
 
-ext = Extension('mc', sources=["mamonca/mc.pyx"], language="c++", extra_compile_args=['-fopenmp'], extra_link_args=['-lgomp'])
+ext = Extension('mc', sources=["mamonca/mc.pyx", "mamonca/cMC.cpp", "mamonca/cMC.h", "mamonca/cMC.pxd"], language="c++", extra_compile_args=['-fopenmp'], extra_link_args=['-lgomp'])
```
Comment (samwaseda) [inline setup.py:17]: I tried to compile my code with this change, but it failed to load `cMC.h`. Did it work for you?

**Comment by jan-janssen** [comment]
The important part for the conda package and the pypi package is, that you first create a source distribution 
```
python setup.py sdist
```
then you navigate to the folder and extract the sources:
```
cd dist
tar -xf mamonca-0.0.3.tar.gz
```
and then in the extracted sources build the python package:
```
cd mamonca-0.0.3
python setup.py build_ext --inplace
```
Currently this fails, with the master branch as well as with the changes in this pull request. But that is the part that has to work for the pypi package and the conda package.

**Comment by jan-janssen** [comment]
@srmnitc Do you have an idea how to fix this?

**Comment by jan-janssen** [comment]
Use https://github.com/samwaseda/mamonca/pull/3 instead

**Pull request closed without merging**